kafka_broker.yml

```diff
@@ -15,7 +15,7 @@
       set_fact:
         install_pattern: "{{ 'parallel' if service_state != 'running' or kafka_broker_deployment_strategy == 'parallel' else 'serial' }}"
```

Comment:
Maybe to make it more readable we could do 
```
        install_pattern: "{{ 'parallel' if service.state != 'running' or kafka_broker_deployment_strategy == 'parallel' else 'serial' }}"
      vars:
        service: "{{ ansible_facts.services[kafka_broker_service_name + '.service'] | default({'state':'unknown'}) }}"
  
```

Reply:
@erikgb wdyt? The issue is some users on python27 are complaining that the default here 
```
"{{ ansible_facts.services[kafka_broker_service_name + '.service'].state | default('unknown') }}"
```
is failing because `ansible_facts.services[kafka_broker_service_name + '.service']` returns nothing, so i guess we need a default on that element before doing the `.state`

Reply:
@domenicbove I do not have a Python 2.7 environment to test in, but I am hoping we can avoid the duplication. 😄 Does the following work?

`service_state: "{{ ansible_facts.services[kafka_broker_service_name + '.service'] | default({})['state'] | default('unknown')}}"`

Reply:
But your suggestion also works - for a single task. But it used in multiple tasks, I would try to stick with a "fact".

Reply:
@erikgb oh ya yours better! @nsharma-git can you test? I dont have python2.7 either. 

@nsharma-git Can you please include the error message from the customer in the description?

Reply:
Thanks @erikgb and @domenicbove . 
I have updated the description to give more context. Since we discovered the sensitive information is getting printed only when we use `diff` flag, I'm checking with security expert, if this is a real concern. Meanwhile, I'll verify  on python27.

Reply:
You should inform your customers about: https://www.python.org/doc/sunset-python-2/. They are running a great risk still on python 2....